biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/GenbankReader.java

```diff
@@ -153,35 +154,56 @@ public GenbankReader(
 		}
 
 		LinkedHashMap<String,S> sequences = new LinkedHashMap<>();
-		@SuppressWarnings("unchecked")
 		int i=0;
 		while(true) {
```

Comment:
This is what is hanging the build. Because the stream returned is null (the file does not exist, see comment below), sequence is always null and this while loop never finishes.

The while loop is not needed anymore, simply take an action if the sequence is null.